test/compilable/cppmangle.d

```diff
@@ -1251,3 +1251,15 @@ extern(C++, `bar`)
 {
     void func19542(T)();
 }
+
+// https://issues.dlang.org/show_bug.cgi?id=20700
+version (Windows) extern(C++)
+{
+    void test20700_1(Struct20700);
+    extern(C++, class) struct Struct20700 {}
+    void test20700_2(Struct20700);
+
+    // Note: Needs to be `BV` (`class`), not `BU` (`struct`)
+    static assert(test20700_1.mangleof == `?test20700_1@@YAXVStruct20700@@@Z`);
+    static assert(test20700_2.mangleof == `?test20700_2@@YAXVStruct20700@@@Z`);
```

Comment:
Where's the `'B'`?

Reply:
Ah, I remove the `const ref`. Will fix the comment. But also, I think I need to address 20701 at the same time. Let me add some more tests.

Reply:
OK, ping me when its good.